src/stdlib.sol

```diff
@@ -62,6 +62,18 @@ abstract contract stdCheats {
         vm_std_cheats.startPrank(who, origin);
     }
 
+    // Allows you to the balance of an account for a majority of tokens
+    // Be careful not to break something!
+    function tip(address to, uint256 give, address which) public {
```

Comment:
Why dont you just use stdstore instead? i imagine there will be more useful things like this that pop up

Reply:
can we change the variable arrangement as well? i think (token, to, give) is most intuitive

Reply:
https://mirror.xyz/brocke.eth/PnX7oAcU4LJCxcoICiaDhq_MUUu9euaM8Y5r465Rd2U < see here. I use stdstore

Reply:
thanks, brock. i didn't know u can use it like that.

the article was informative!

Reply:
it's ready, ser @brockelmore